src/client/Microsoft.Identity.Client/Cache/CacheTypeUsed.cs

```diff
@@ -17,10 +17,10 @@ public enum CacheTypeUsed
         /// <summary>
         /// Specifies if the L1 cache is used
         /// </summary>
-        L1Cache,
+        L1Cache = 0,
```

Comment:
Should there be a None value ?

Reply:
Lets have default as None for scenario when no cache is used.